src/smart-components/group/group.js

```diff
@@ -28,6 +28,7 @@ const Group = ({
   isFetching,
   onDelete,
 }) => {
+  const isPlatformDefault = !(uuid !== 'default-access');
```

Comment:
No need to have two negative comparators
```suggestion
  const isPlatformDefault = uuid === 'default-access';
```

Reply:
I will make this change, but I guess it was unconscious choice I always try to check `!==` rather than `===` because the former is little more efficient, but I guess it doesn't make much difference here